xm-commons-timeline/src/main/java/com/icthh/xm/commons/timeline/config/TimelineInterceptor.java

```diff
@@ -97,7 +106,38 @@ private boolean isIgnoredRequest(HttpServletRequest request) {
                 }
             }
         }
-        return false;
+
+        String responseBody = getResponseContent(response);
```

Comment:
Please add check: isNotEmpty(ignoredTypeKeys) before parsing http response. Motivation: if we don't have configured ignored types keys parsing response body operation was redundant